tests/phpunit/api/v3/CaseTest.php

```diff
@@ -163,8 +163,13 @@ public function testCaseUpdate() {
     $params['subject'] = $case['subject'] = 'Something Else';
     $this->callAPISuccess('case', 'create', $params);
 
-    // Verify that updated case is exactly equal to the original with new subject.
+    // Verify that updated case is equal to the original with new subject.
     $result = $this->callAPISuccessGetSingle('Case', array('case_id' => $id));
+    // Modification dates are likely to differ by 0-2 sec. Check manually.
+    $this->assertTrue($result['modified_date'] >= $case['modified_date']);
```

Comment:
I saw you accepted my suggestion to use `assertLessThan()` so maybe can you use `assertGreaterThanOrEqual` here, especially because there's no custom message for this assertion